cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -471,23 +471,23 @@ function LogViewerController ($scope, LogViewerStore, myLogsApi, LOGVIEWERSTORE_
 
     this.rawUrl = url;
 
-    myLogsApi.getLogsStartAsJson({
+    myLogsApi.nextLogsJsonOffset({
         namespace : this.namespaceId,
         appId : this.appId,
         programType : this.programType,
         programId : this.programId,
         runId : this.runId,
-        start : this.startTimeSec
+        fromOffset: -1 + '.' + this.startTimeSec*1000
```

Comment:
This cannot be `-1` all the time right? We need to use the offset from the last log that we got from the backend to get the next set of logs right?